src/events.rs

```diff
@@ -49,10 +49,9 @@ pub const CLAP_NOTE_EXPRESSION_VOLUME: clap_note_expression = 0;
 pub const CLAP_NOTE_EXPRESSION_PAN: clap_note_expression = 1;
 pub const CLAP_NOTE_EXPRESSION_TUNING: clap_note_expression = 2;
 pub const CLAP_NOTE_EXPRESSION_VIBRATO: clap_note_expression = 3;
+pub const CLAP_NOTE_EXPRESSION_EXPRESSION: clap_note_expression = 5;
 pub const CLAP_NOTE_EXPRESSION_BRIGHTNESS: clap_note_expression = 4;
```

Comment:
I think these values are swapped (see https://github.com/free-audio/clap/blob/202c01d845eae72f2f9a1f6a4c4654b7b1145102/include/clap/events.h#L117-L118).

Reply:
Good catch. They changed order in CLAP 0.21 but I forgot to also swap the values around.